### ISSUE TITLE
Another instance

### ISSUE THREAD

**Issue by pierreozoux** [Laverna/laverna#230]
Hi,

as an [IndieHoster](https://indiehosters.net), I'm happy to announce you that I started to host a "shared instance" at: https://laverna.tokoloho.co/

But, I have to admit, I don't really get why it is important for people to have a personal instance? I thought that you "install" an unhosted app by visiting a website. So it doesn't really matter to have your personal instance? But maybe I'm wrong?

cc @michielbdejong

Do you think necessary this instance? I'm doubting now :)

Anyway, thanks for your project, and if you need hosting, please let me know, I'd be happy to help!

**Comment by michielbdejong** [comment]
Indeed, I think Laverna is statics-only. Having it mirrored in more places helps its resilience, but does not bring anything personal to it, in fact, the user(s) of the domain tokoloho.co might as well use an instance unrelated to their identity.

If you click the red pen icon, and then save, a 'settings' button appears on the top right. There, under 'Cloud', you can connect remoteStorage. So that's where people's personal server comes in, as a user of Laverna you want to use remoteStorage to save notes on your own personal server (and an IndieHoster can manage that remoteStorage server for you).

**Comment by pierreozoux** [comment]
Yes, it is static only. I understand the resilience part, but IMHO, I would find it confusing for a user to know that there are different instances. On top of that, as a user, I don't know if it is the last version, if the second instance is not trying to fool me.

And at the end, I can't switch from one instance to another if I use just localstorage.

I think it is confusing to say to people they should use their own instance. It is the same as I am a debian user, I use debian repo, but the guy at the debian repo tell me, "ok, you can use the public repo, but we recommand you to have your own.". As a debian user, I would be frustrated. I guess the guys behind this repo use Gentoo :)

One more time, maybe I'm wrong, but I find it confusing :)
